src/main/java/frc/robot/subsystems/ShooterFlywheel.java

```diff
@@ -69,11 +69,11 @@ public ShooterFlywheel() {
         motorTwo.setIdleMode(IdleMode.kCoast);
         motorTwo.setSmartCurrentLimit(40, 40);
         motorTwo.setInverted(false);
-        motorTwo.getPIDController().setP(0.001);
-        motorTwo.getPIDController().setI(0.0001);
+        motorTwo.getPIDController().setP(0.002);
+        motorTwo.getPIDController().setI(0.001);
         motorTwo.getPIDController().setD(0);
         motorTwo.getPIDController().setFF(0.000152);
-        motorTwo.getPIDController().setIZone(0.0015);
+        motorTwo.getPIDController().setIZone(1);
```

Comment:
It'd be really nice if you had a method which did this automatically from something like a `PIDConfig` (maybe it's called `PIDFConfig`? I forget)

Reply:
Unfortunately unless I missed it, it is not a functionality it has.